dendrite-config.yaml

```diff
@@ -0,0 +1,73 @@
+# The config file version format
+version: v0
+
+# The matrix specific config
+matrix:
+    # The name of the server. This is usually the domain name, e.g 'matrix.org', 'localhost'.
+    server_name: "example.com"
+    # The path to the PEM formatted matrix private key.
+    private_key: "/var/dendrite/matrix_key.pem"
```

Comment:
I've moved the keys to /etc 3a212a4.
I was using /var to match the example media-repo config.
(Assuming the docker compose stuff lands in #140 we would probably be better of assuming that the config was in a /etc directory rather than a /home directory)